src/main.js

```diff
@@ -67,6 +67,7 @@ export default function createApp({
 	app.provide('kvAuth0', kvAuth0);
 	app.provide('locale', locale);
 	app.provide('$kvTrackEvent', app.config.globalProperties.$kvTrackEvent); // provide kvTrackEvent for composition api
+	app.provide('$appConfig', appConfig); // provide appConfig for composition api
```

Comment:
Does this also cover the use case below for components? Seems like it might?

Reply:
How will this manifest in the compiled app? Maybe a globally loaded file?

Reply:
I just tested locally, and provide properties aren't added to the `this` object for components. My guess is that we could convert `this.` usages to inject, if we desired at some point.

Regarding where it exists when compiled, it doesn't look like it's any different than the exist use of the app config. The config is passed to `createApp` when the app starts, and it seems like the data remains in memory for the components. I tried compiling the app, and I'm not seeing app config props being defined, only used.